src/styles/_catalog.scss

```
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
```

```suggestion
    height: 300px;
    margin-bottom: 48px;
    
    border: 1px solid #ccc;
    border-radius: 5px

```
Margin-bottom must be the part of block-properties, move it before border and add between margin and border blank line, they are different groups

---

Thanks, I didn't notice that